VimeoNetworking/VimeoNetworkingTests/NSErrorExtensionTests.swift

```diff
@@ -60,11 +60,28 @@ class NSErrorExtensionTests: XCTestCase {
         XCTAssertEqual(self.testApiError.vimeoDeveloperMessage, "The parameters passed to this API endpoint did not pass Vimeo\'s validation. Please check the invalid_parameters list for more information")
     }
     
-    func testVimeoServerErrorCode() {
+    func testVimeoServerErrorCodeInJSONErrorBody() {
         XCTAssertNotNil(self.testApiError.vimeoServerErrorCode)
         XCTAssertEqual(self.testApiError.vimeoServerErrorCode!, 2204)
     }
     
+    func testLegacyVimeoServerErrorCode() {
+        var userInfo = self.testApiError.userInfo
+        userInfo["VimeoErrorCode"] = 4101
```

Comment:
it's a private constant in the NSError+Extensions.swift file.  I don't know if the @testable keyword works with class extensions.